CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+### Features
+
+- The filesystem adapters for the `sentry` driver now extend the well-known Laravel classes they decorate,
+  `Illuminate\Filesystem\FilesystemAdapter` and `Illuminate\Filesystem\AwsS3V3Adapter`.
+
+  Enabling the feature can be simplified by wrapping the configuration for your disks
+  with a call to `Sentry\Laravel\Features\Storage\Integration::withSentryDriver()`
+  in your `config/filesystems.php` file.
+
+  ```php
+  'disks' => Sentry\Laravel\Features\Storage\Integration::withSentryDriver([
+      'local' => [
+          'driver' => 'local',
+          'root' => storage_path('app'),
+          'throw' => false,
+      ],
+
+      ...
+  ]),
```

Comment:
Do I have to wrap all my storages in sentry? Would it be an option wrap single drivers in sentry? Like I only want to have breadcrumbs of my S3 and ignore my local driver.
This could also help with the problem of specific classes expected by people. If I really need that specific AbcAdaper I can leave that as is but still track the other drivers with sentry.